src/Symfony/Component/Serializer/Encoder/CsvEncoder.php

```diff
@@ -21,6 +21,10 @@
 class CsvEncoder implements EncoderInterface, DecoderInterface
 {
     const FORMAT = 'csv';
+    const DELIMITER_KEY = 'csv_delimiter';
```

Comment:
Just asking: is there any reason to name these constants `*_KEY` instead of `*_CHAR`? I was checking the popular CSV library published by ThePhpLeague and they say that the separator, enclosure, etc. can only be 1 char: https://github.com/thephpleague/csv/blob/master/src/AbstractCsv.php#L42